eq-author/src/components/buttons/ToggleSwitch/index.js

```diff
@@ -21,6 +21,11 @@ const backgroundSize = {
   height: 1,
 };
 
+const labelColors = {
+  on: colors.black,
+  off: colors.darkGrey,
```

Comment:
off toggle label needs to be colors.grey not colors.Darkgrey
that was my mistake!

Reply:
Cheers for the heads up! I've merged in your changes, so this is fixed.